src/main/scala/com/ironcorelabs/UserCreateKeyPair.scala

```diff
@@ -0,0 +1,18 @@
+package com.ironcorelabs.scala.sdk
+
+import com.ironcorelabs.{sdk => jsdk}
+
+case class UserCreateKeyPair(
```

Comment:
I think we should change this name. UserCreateResult seems best. We should change it all the way down to ironoxide IMO.

Reply:
Changed to `UserCreateResult`